packages/ui-keyring/src/stores/File.ts

```diff
@@ -25,12 +25,16 @@ export default class FileStore implements KeyringStore {
       .readdirSync(this.#path)
       .filter((key): boolean => !['.', '..'].includes(key))
       .forEach((key): void => {
-        cb(key, this._readKey(key));
+        if(this._readKey(key)) {
+          cb(key, this._readKey(key));
+        }
       });
   }
 
   public get (key: string, cb: (value: KeyringJson) => void): void {
-    cb(this._readKey(key));
+    if(this._readKey(key)) {
+      cb(this._readKey(key));
+    }
```

Comment:
```suggestion
    const value = this._readKey(key);
    
    value && cb(value);
```

Reply:
The rule states that I cannot return undefined values